changelogs/current.yaml

```diff
@@ -2,6 +2,13 @@ date: Pending
 
 behavior_changes:
 # *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+- area: eds
+  change: |
+    Introduced caching of EDS assignments when used with ADS. Without this change, Envoy required EDS assignments to always
+    be sent after a cluster was updated. Following a cluster upadate, Envoy will wait for an EDS assignment until
+    :ref:`initial_fetch_timeout <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout` times out, and then apply
```

Comment:
Hmm... CI complains that the doc label is undefined: https://dev.azure.com/cncf/envoy/_build/results?buildId=145343&view=logs&j=9fd58ae0-0c50-56b1-f517-c2b9afda9c9b&t=c8fc762c-55bb-57d4-4cab-8ec527e7a423&l=117.

@phlax not sure why this is so, IIUC the [field](https://github.com/envoyproxy/envoy/blob/35684d731a2efc5a3415be191e8a8944539ccf4f/api/envoy/config/core/v3/config_source.proto#L248C3-L248C3) is correctly referenced. Am I missing anything?

Reply:
i can see 

```rst
.. _envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout:
```

here https://www.envoyproxy.io/docs/envoy/latest/_sources/api-v3/config/core/v3/config_source.proto.rst.txt

so ref looks correct